code/modules/mentor/mentor_verbs.dm

```diff
@@ -12,3 +12,27 @@ GLOBAL_PROTECT(mentor_verbs)
 /client/proc/remove_mentor_verbs()
 	remove_verb(GLOB.mentor_verbs)
 	reset_badges()
+
+/client/verb/mrat()
```

Comment:
This needs to consider people being muted from mentorhelping and should probably be a config option to enable.